pipeline/bicleaner/download_pack.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+"""
+Downloads bicleaner-ai model for a lanuage pair.
+Fallbacks to the multilingual model if the lanuage pair is not supported.
+
+Example:
+    python download_pack.py \
+        --src=en \
+        --trg=ru \
+        --compression_cmd=zstd \
+        artifacts/bicleaner-model-en-ru.zst
+"""
+
+import argparse
+import os
+import shutil
+import subprocess
+import tarfile
+import tempfile
+from typing import Optional
+
+
+def _run_download(src: str, trg: str, dir: str) -> subprocess.CompletedProcess:
+    return subprocess.run(
+        ["bicleaner-ai-download", trg, src, "full", dir], capture_output=True, check=False
+    )
+
+
+def _compress_dir(dir_path: str, compression_cmd: str) -> str:
+    print(f"Compressing {dir_path}")
+    if compression_cmd not in ["gzip", "zstd", "zstdmt", "pigz"]:
+        raise ValueError("Unsupported compression tool.")
+
+    tarball_path = dir_path + ".tar"
```

Comment:
nit:
```suggestion
    tarball_path = f"{dir_path}.tar"
```

Reply:
nit: Here and in a few other places, string concatenation with "+" is not very Pythonic